emscripten-simulated-jerry-api/print.js

```diff
@@ -0,0 +1,20 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
```

Comment:
This file and the pre-js linker flag to include it, should also be removed now that print() is part of jerry ext.